sip-servlets-client/src/main/java/org/mobicents/javax/servlet/sip/dns/DNSResolver.java

```diff
@@ -60,7 +61,7 @@
 	 * @param uri the uri used to find the corresponding SipURI
 	 * @return the SipURI found through ENUM methods or the uri itself if the uri is already a SipURI without a user=phone param
 	 */
-	SipURI getSipURI(URI uri);
+	SipURI getSipURI(URI uri, SipSession sipSession);
```

Comment:
Ok, then let me go through the changes and i will submit a PR against your
fork, so you can merge from there.

Not sure when this will be ready, since im up to other things, i will try
tomorrow,ok?

On Thu, Jun 23, 2016 at 1:43 AM, Luu Huu Hoan <notifications@github.com>
wrote:

> In
> sip-servlets-client/src/main/java/org/mobicents/javax/servlet/sip/dns/DNSResolver.java
> <https://github.com/RestComm/sip-servlets/pull/163#discussion_r68155260>:
>
> > @@ -60,7 +61,7 @@
> >  	 * @param uri the uri used to find the corresponding SipURI
> >  	 * @return the SipURI found through ENUM methods or the uri itself if the uri is already a SipURI without a user=phone param
> >  	 */
> > -	SipURI getSipURI(URI uri);
> > +	SipURI getSipURI(URI uri, SipSession sipSession);
>
> It would be perfect if you help me on thread local. I will use your api to
> implement my code.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RestComm/sip-servlets/pull/163/files/80fac53a2680274d75ce7f27da19ae5f4f488f82#r68155260>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHe9Scyu2QtAkH2_SfYEnwwvydzSL_Czks5qOciTgaJpZM4Iz60n>
> .
>